calcom/cal.com
# fix: Enhance the Robustness of Embed Snippets for Multiple Embeds on a Single Page

## What does this PR do?

Embed Snippets are now using namespaces so that no change is required when installing multiple embed snippets on a single page.
Because event slug is unique and changes less frequently, it is being used as a namespace.

Fixes #10031

## Type of change

## How should this be tested?
- Create an event-type
- Click Embed
- Select any embed type
- Observe the uses of ns.${eventSlug} everywhere.
- Install the snippet as per instructions and test that they are working.

## Mandatory Tasks


## Checklist
- I haven't added tests that prove my fix is effective or that my feature works
